ironfish/src/rpc/routes/peers/getPeerMessages.ts

```diff
@@ -16,6 +16,7 @@ type PeerMessage = {
         type: string
         payload: Record<string, unknown>
       }
+    | Buffer
```

Comment:
The `yup` validator in this file might have to be updated to accept some form of `Buffer` -- I'm a little surprised TS didn't complain about needing to update the `yup` validation, interesting

Reply:
It seems like the schema [isn't used](https://github.com/iron-fish/ironfish/blob/master/ironfish/src/rpc/routes/peers/getPeerMessages.ts#L62). Will take a look in a subsequent PR.